src/components/storage/ModifyStorage.jsx

```diff
@@ -41,12 +41,12 @@ export const ModifyStorage = ({ idPrefix, onCritFail, onRescan, setShowStorage,
               variant="link"
               icon={<WrenchIcon />}
               onClick={() => {
-                  window.localStorage.setItem("cockpit_anaconda",
-                                              JSON.stringify({
-                                                  mount_point_prefix: targetSystemRoot,
-                                                  available_devices: selectedDevices,
-                                                  efi: isEfi,
-                                              })
+                  window.sessionStorage.setItem("cockpit_anaconda",
+                                                JSON.stringify({
+                                                    mount_point_prefix: targetSystemRoot,
+                                                    available_devices: selectedDevices,
+                                                    efi: isEfi,
```

Comment:
This should also set `"default_fsys_type"`.

Reply:
I really dont know where to parse it from. I dont think there is an API for that currently. I prefer to figure it out in followup.